scrape.py

```diff
@@ -57,6 +58,7 @@ def generate_ics_file(uid, data):
         event.add('summary', 'Gelber Sack Abholtermin')
         event.add('dtstart', start)
         event.add('dtend', end)
+        event.add('uid', datetime.now().strftime("%Y%m%dT%H%M%S-%f") + "@" + gethostname())
```

Comment:
I would prefer to use a UUID here, e.g. `event.add('uid', str(uuid.uuid4()) + "@" + gethostname())`

The current datetime (even down to miliseconds) is less likely to be unique.